sdk/communication/Azure.Communication.Sms/src/SmsClient.cs

```diff
@@ -132,9 +134,9 @@ public virtual Response<SmsSendResult> Send(string from, string to, string messa
         {
             Argument.AssertNotNullOrEmpty(from, nameof(from));
             Argument.AssertNotNullOrEmpty(to, nameof(to));
-            Response<IEnumerable<SmsSendResult>> allSmsSendResults = Send(from, new[] { to }, message, options, cancellationToken);
-            SmsSendResult sendSMSResult = allSmsSendResults.Value.First();
-            return Response.FromValue(sendSMSResult, allSmsSendResults.GetRawResponse());
+            Pageable<SmsSendResult> allSmsSendResults = Send(from, new[] { to }, message, options, cancellationToken);
+            Page<SmsSendResult> firstPage = allSmsSendResults.AsPages().First();
+            return Response.FromValue(firstPage.Values[0], firstPage.GetRawResponse());
```

Comment:
`firstPage.Values[0]`, would there be chance of out of bound exception here?

Reply:
That would be a bug in the service so I do not think we should account for it here.